src/databricks/labs/ucx/hive_metastore/migration_status.py

```diff
@@ -127,8 +127,11 @@ def _crawl(self) -> Iterable[MigrationStatus]:
             yield table_migration_status
 
     def _try_fetch(self) -> Iterable[MigrationStatus]:
-        for row in self._fetch(f"SELECT * FROM {self._schema}.{self._table}"):
-            yield MigrationStatus(*row)
```

Comment:
this change will break other things. the fix has to be in `LocalCheckoutContext.linter_context_factory`

Reply:
Done